package.json

```diff
@@ -9,6 +9,8 @@
     "data-uri-to-blob": "0.0.4",
     "debounce": "~1.0.0",
     "drag-reorderable": "~0.1.0",
+    "events": "~1.1.1",
+    "fast-levenshtein": "~1.0.7",
```

Comment:
Yeah i had a pain with that line in rebase, will fix before moving out of WiP state